src/Compilers/CSharp/Portable/Symbols/Source/SourceNamespaceSymbol.cs

```diff
@@ -120,7 +119,7 @@ public override ImmutableArray<SyntaxReference> DeclaringSyntaxReferences
         {
             get
             {
-                return ComputeDeclaringReferencesCore();
+                return IsInjected ? ImmutableArray<SyntaxReference>.Empty : ComputeDeclaringReferencesCore();
```

Comment:
>return IsInjected ? ImmutableArray<SyntaxReference>.Empty : ComputeDeclaringReferencesCore(); [](start = 16, length = 93)

This change looks redundant since we modify ```ComputeDeclaringReferencesCore``` too.